README.md

```diff
@@ -0,0 +1,67 @@
+# Conduit Connector Vitess
+
+## General
+
+The [Vitess](https://vitess.io/) connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides both, a source and a destination Vitess connector.
+
+### Prerequisites
+
+- [Go](https://go.dev/) 1.18
+- (optional) [golangci-lint](https://github.com/golangci/golangci-lint) 1.48.0
```

Comment:
I don't think we need any specific version, and since golangci-lint has releases relatively frequently, we probably don't need to put a version at all (so we avoid being outdated).

Reply:
Done